int8 TAKE_OFF_AND_HOLD = 1
int8 HOLD = 2
int8 MISSION_EXECUTION = 3
int8 AUTO_LAND = 4
int8 DEFAULT_TIMEOUT = 90
#request fields
int8 mode
float64 sub
int8 timeout 
---
#response constants
bool success
int8 result